### DISCUSSION TITLE
Search queries - combine "and" and "or" logic

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#8870] [Ideas]
To use "and", we make query like this:
`{{craft.entries.search('field1:something1 field2:something2')}}`

To use "or", we make query like this:
`{{craft.entries.search('field1:something1 OR field2:something2')}}`

We however cannot make a query like this:
`(something or something2) and (something3 or something4)`

Such query would be useful in case of filtering by values of multiple checkbox fields.

**Comment by GaryReckard** [comment]
This would be very useful. Is there another way to accomplish this at the moment?

**Comment by luke-nehemedia** [comment]
I agree, this would be very useful indeed!

**Comment by piotrpog** [comment]
Any plans to have this functionality included?
